ex/high_throughput_inference/standalone_workermanager.py

```diff
@@ -0,0 +1,44 @@
+# isort: off
+import dragon
+from dragon import fli
+from dragon.channels import Channel
+from dragon.data.ddict.ddict import DDict
+from dragon.utils import b64decode, b64encode
+from dragon.globalservices.api_setup import connect_to_infrastructure
+# isort: on
+import argparse
+import os
+
+
+from smartsim._core.mli.comm.channel.dragonchannel import DragonCommChannel
+from smartsim._core.mli.infrastructure.worker.torch_worker import TorchWorker
+from smartsim._core.mli.infrastructure.control.workermanager import (
+    WorkerManager,
+)
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser("Worker Manager")
+    parser.add_argument("--device", default="gpu")
+    args = parser.parse_args()
+    connect_to_infrastructure()
+    ddict_str = os.environ["SS_DRG_DDICT"]
+    ddict = DDict.attach(ddict_str)
+
+    to_worker_channel = Channel.make_process_local()
+    to_worker_manager_channel = Channel.make_process_local()
+    channels = [Channel.make_process_local() for _ in range(100)]
+    to_worker_fli = fli.FLInterface(main_ch=to_worker_channel, manager_ch=to_worker_manager_channel, stream_channels=channels)
```

Comment:
Putting a placeholder here so we make a ticket to look at sender-supplied channels.

Reply:
Actually, I looked into that and updated it. It now works with sender-supplied channels.